libs/core/langchain_core/runnables/graph.py

```diff
@@ -111,35 +132,25 @@ class MermaidDrawMethod(Enum):
     API = "api"  # Uses Mermaid.INK API to render the graph
 
 
-def node_data_str(node: Node) -> str:
+def node_data_str(id: str, data: Union[Type[BaseModel], RunnableType]) -> str:
```

Comment:
might be worth switching all these to be `_` prefixed to make clear they're internal